plugins/nodes/src/js/pages/nodes/nodes-table/NodesTableContainer.js

```diff
@@ -87,4 +126,18 @@ class NodesTableContainer extends mixin(StoreMixin, QueryParamsMixin) {
   }
 }
 
-module.exports = NodesTableContainer;
+const pollingInterval$ = timer(0, Config.getRefreshRate());
+const nodes$ = pollingInterval$.pipe(
```

Comment:
Let's use the DL here as discussed offline, so with a simple networks resolver

Reply:
Refactored the NodesResolver to be NodesNetworkResolver